### PR TITLE
Lint the code base and move to a consistent yaml format

### PR DESCRIPTION
**Pull request by DanyC97**
@arilivigni  please help out review it.

### CODE REVIEW

**Comment by DanyC97** [comment]
@tdawson fyi.

File: playbooks/openshift/roles/bfs/tasks/clone_from_github.yml [hunk 1]
```diff
@@ -11,4 +15,5 @@
     clone: yes
     update: yes
     accept_hostkey: yes
-  when: dot_git_path.stat.isdir is undefined or dot_git_path.stat.isdir == False
+  when: (dot_git_path.stat.isdir is undefined) or
```
Comment (arilivigni) [inline playbooks/openshift/roles/bfs/tasks/clone_from_github.yml]: Why did you decide to use parens here and not elsewhere?

File: playbooks/openshift/roles/bfs/tasks/build_openshift_srpm.yml [hunk 2]
```diff
@@ -22,13 +23,17 @@
...
+    - git_log.stdout == ""
+    - not bleeding_edge
 
 - name: tag origin version
   shell: git tag -d {{ origin_version }}
   args:
     chdir: "{{ openshift_repo_path }}"
-  when: (git_log.stdout == "" and not bleeding_edge)
+  when:
+    - git_log.stdout == "" and
```
Comment (arilivigni) [inline playbooks/openshift/roles/bfs/tasks/build_openshift_srpm.yml]: You didn't use parens here?

File: playbooks/openshift/roles/bfs/tasks/build_openshift_srpm.yml [hunk 3]
```diff
@@ -22,13 +23,17 @@
   shell: git checkout {{ origin_version }}
   args:
     chdir: "{{ openshift_repo_path }}"
-  when: (git_log.stdout == "" and not bleeding_edge)
+  when:
+    - git_log.stdout == ""
```
Comment (arilivigni) [inline playbooks/openshift/roles/bfs/tasks/build_openshift_srpm.yml]: You didn't use parens here

File: playbooks/openshift/roles/bfs/tasks/build_openshift-ansible_srpm.yml [hunk 4]
```diff
@@ -33,7 +34,9 @@
   shell: "git checkout {{ oa_version }}"
   args:
     chdir: "{{ openshift_repo_path }}"
-  when: (git_log.stdout == "" and bleeding_edge|bool == false)
+  when:
+    - git_log.stdout == ""
```
Comment (arilivigni) [inline playbooks/openshift/roles/bfs/tasks/build_openshift-ansible_srpm.yml]: you didn't use parens here?

File: playbooks/openshift/roles/bfs/tasks/clone_from_github.yml (hunk 1 above)
Comment (DanyC97) [reply playbooks/openshift/roles/bfs/tasks/clone_from_github.yml]: i find it easier to use parens to put in the spot light the `or` condition due to the fact that ansible doesn't let me do something like this

```
  when: 
    - dot_git_path.stat.isdir is undefined or
    - dot_git_path.stat.isdir == False
```
nor
```
  when: 
    - (dot_git_path.stat.isdir is undefined) or
    - (dot_git_path.stat.isdir == False)
```
however is a different story for  `and` condition where when you can do 

```
  when:
    - git_log.stdout == ""
    - bleeding_edge | bool == false
```
where `and` is implicit and you can't miss it

File: playbooks/openshift/roles/bfs/tasks/build_openshift_srpm.yml (hunk 2 above)
Comment (DanyC97) [reply playbooks/openshift/roles/bfs/tasks/build_openshift_srpm.yml]: the parens for `and` i'd normally use when we have multiple conditions which must be satisfied together, like

```
when:
   - (foo and foo is defined) or
   - bar == 'orange'
```

Comment (DanyC97) [reply playbooks/openshift/roles/bfs/tasks/build_openshift_srpm.yml]: and i just spotted a bug which i'm going to fix it in a bit.

File: playbooks/openshift/roles/bfs/tasks/build_openshift_srpm.yml (hunk 3 above)
Comment (DanyC97) [reply playbooks/openshift/roles/bfs/tasks/build_openshift_srpm.yml]: see above

File: playbooks/openshift/roles/bfs/tasks/build_openshift-ansible_srpm.yml (hunk 4 above)
Comment (DanyC97) [reply playbooks/openshift/roles/bfs/tasks/build_openshift-ansible_srpm.yml]: see above

**Pull request merged**